package.json

```diff
@@ -185,7 +185,7 @@
     "time-grunt": "1.4.0"
   },
   "scripts": {
-    "postinstall": "./node_modules/.bin/bower install --allow-root",
+    "postinstall": "./node_modules/.bin/bower install --allow-root --production",
```

Comment:
this is out of scope of this PR, please revert it

Reply:
He separated them into two commits.

Reply:
it's still out of scope of this PR

Reply:
ok. So @ThanhxD please separate them into two pull request.